.github/workflows/tests.yml

```diff
@@ -419,6 +419,7 @@ jobs:
 
   static-site:
     needs: process-results
+    if: always() && github.ref == 'refs/heads/main' && github.repository == 'coiled/coiled-runtime'
```

Comment:
```suggestion
    # Always generate the site, as this can be skipped even if an indirect dependency fails (like a test run)
    if: always() && github.ref == 'refs/heads/main' && github.repository == 'coiled/coiled-runtime'
```